modules/ivc_champva/app/jobs/ivc_champva/missing_form_status_job.rb

```diff
@@ -74,20 +73,20 @@ def send_failure_email(form, template_id)
       end
     end
 
-    # Fires off a notification email to Pega so they know a user has been
-    # notified of a failed submission.
+    # Fires off a notification email to Pega so they know the communication status of
+    # submissions with missing Pega statuses.
     #
     # @param form_data [hash] hash of form details (see `send_failure_email`)
     # @param form [IvcChampvaForm] form object in question
-    def send_zsf_notification_to_pega(form)
-      form_data = construct_email_payload(form, 'PEGA-TEAM-ZSF')
+    def send_zsf_notification_to_pega(form, template_id)
+      form_data = construct_email_payload(form, template_id)
       form_data = form_data.merge({
                                     email: Settings.vanotify.services.ivc_champva.pega_inbox_address
                                   })
       if IvcChampva::Email.new(form_data).send_email
-        monitor.track_send_zsf_notification_to_pega(form_data[:form_uuid])
+        monitor.track_send_zsf_notification_to_pega(form_data[:form_uuid], template_id)
       else
-        monitor.track_failed_send_zsf_notification_to_pega(form_data[:form_uuid])
+        monitor.track_failed_send_zsf_notification_to_pega(form_data[:form_uuid], template_id)
       end
     end
```

Comment:
Adjusted this so it takes a template_id, allowing us to reuse for the 2-day advance notice notification email.